runatlantis.io/docs/automerging.md

```diff
@@ -29,6 +29,20 @@ Automerging can be enabled either by:
 If automerge is enabled, you can disable it for a single `atlantis apply`
 command with the `--auto-merge-disabled` option.
 
+## How to set merge method for automerge
+
+If automerge is enabled, you can use `--merge-method` option
+for `atlantis apply` command to specify which merge method use.
+
+```shell
+atlantis apply --merge-method squash
+```
+
+Implemented only for GitHub. You can choose one of them:
+- merge
+- rebase
+- squash
```

Comment:
Suggestion:

```shell
atlantis apply -- --automerge-method <method>
```

The `method` must be one of:

- `merge`
- `rebase`
- `squash`

This is currently only implemented for the GitHub VCS.

Reply:
Thanks! It would be nice to rename it. But I suppose this flag should be called `auto-merge-method`. I think the best way to keep consistency between flags in the apply command is to use the same prefix that is already used in `auto-merge-disabled` ([How to Disable](https://www.runatlantis.io/docs/automerging.html#how-to-disable))

Reply:
fixed in https://github.com/runatlantis/atlantis/pull/4895/commits/cf878039e12effa3f377c3d4076ee519ea2184ff

Reply:
Thanks @a1k0u can you update the PR title and description with the new flag name.

Reply:
👌
I've updated